README.md

```diff
@@ -2,35 +2,29 @@
 
 Welcome! The open-cluster-management.io project is focused on enabling end-to-end visibility and control across your Kubernetes clusters.
 
-The open-cluster-management architecture uses a hub - agent model. The hub centralizes control of all the managed clusters. An agent, which we call the klusterlet, resides on each managed cluster to manage registration to the hub and run instructions from the hub.
+The Open Cluster Management (OCM) architecture uses a hub - agent model. The hub centralizes control of all the managed clusters. An agent, which we call the klusterlet, resides on each managed cluster to manage registration to the hub and run instructions from the hub.
 
+you can use the [clusteradm CLI](https://github.com/open-cluster-management-io/clusteradm) to bootstrap a control plane for multicluster management. The following diagram illustrates the deployment architecture for the Open Cluster Management.
 
 ![image](assets/ocm-arch.png)
 
 There are a number of key use cases that are enabled by this project, and are categorized to 3 sub projects.
 
 ### Cluster Lifecycle: Cluster registration and management
 
-The API and controllers provide the function for cluster registration, manifests delivery, cluster scheduling and addon management. Simple core functions connect clusters, such as the klusterlet, to the hub. Other components run on this base. The following repositories describe the API and controllers:
+OCM has a group of [APIs](https://github.com/open-cluster-management-io/api) to provide the foundational functions
+in multiple cluster management. 
 
-* https://github.com/open-cluster-management-io/api
-* https://github.com/open-cluster-management-io/registration
-* https://github.com/open-cluster-management-io/work
-* https://github.com/open-cluster-management-io/placement
-* https://github.com/open-cluster-management-io/registration-operator
-* https://github.com/open-cluster-management-io/addon-framework
+The journey of cluster management starts with [Cluster Registration](https://github.com/open-cluster-management-io/registration) which follows a `double opt-in` protocol to establish a MTLS connection from the agent on the managed cluster (Klusterlet) to the hub (Cluster Manager). After this, users or operands on the hub can declare [`ManifestWorks`](https://github.com/open-cluster-management-io/work) which contains a slice of kuebrnetes resource manifests to be distributed and applied to a certain managed cluster. To schedule workloads to a certain set of clusters, users can also declare a [`placement`](https://github.com/open-cluster-management-io/placement) on the hub to dynamically select a set of clusters with certain criterias.
```

Comment:
```suggestion
The journey of cluster management starts with [Cluster Registration](https://github.com/open-cluster-management-io/registration) which follows a `double opt-in` protocol to establish a MTLS connection from the agent on the managed cluster (Klusterlet) to the hub (Cluster Manager). After this, users or operands on the hub can declare [ManifestWorks](https://github.com/open-cluster-management-io/work) which contains a slice of Kubernetes resource manifests to be distributed and applied to a certain managed cluster. To schedule workloads to a certain set of clusters, users can also declare a [Placement](https://github.com/open-cluster-management-io/placement) on the hub to dynamically select a set of clusters with certain criteria.
```
Lets show links with just [xyz] instead of [`xyz`] 
Some other minor nits like spelling.